app/views/support_tickets/thank_you.html.erb

```diff
@@ -1,23 +1,38 @@
-<% content_for :title,  'Thank you for completing this form' %>
+<% content_for :title,  'Support request complete' %>
 
 <%- content_for :before_content do %>
   <% breadcrumbs([{ 'Home' => guidance_page_path },
                   {t('landing_pages.get_support.start') => support_ticket_path},
-                  'Thank you for completing this form',
+                  'Support request complete',
```

Comment:
```suggestion
                  'Support request sent',
```

How about sent/delivered rather than complete? cc @Lloyd-K

Reply:
I like that!